xml/decoders/MTH_PS3_Full.xml

```diff
@@ -34,7 +34,7 @@
         <decVal/>
         <label>PWM max voltage</label>
         <label xml:lang="it">PWM Volt Massimi (0-255):</label>
-        <label xml:lang="de">PWM HÃ¶chstgeschwindigkeit</label>
+        <label xml:lang="de">PWM Hachstgeschwindigkeit</label>
```

Comment:
You should be able to place a proper ``ö`` character here in a valid utf-8 encoded XML file (and ``é`` for French). 
``a`` is not a good substitution; if accented char won't pass, use ``oe`` here.